iati/core/rulesets.py

```diff
@@ -164,10 +161,7 @@ def _valid_rule_configuration(self, case):
             ValueError: When the case is not valid for the type of Rule.
 
         """
-        try:
-            ruleset_schema_section = self._ruleset_schema_section()
-        except AttributeError:
-            raise
+        ruleset_schema_section = self._ruleset_schema_section()
```

Comment:
Since it's only used once and doesn't add a lot to verbosity, it may be clearer to call `seld._ruleset_schema_section()` directly in the `jsonschema.validate()` call.